docs/en/credential-issuance-l2plus.rst

```diff
@@ -0,0 +1,671 @@
+
+Nonce Lifecycle Management
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Each step in the eID Substantial Authentication with MRTD Verification flow MUST use unique nonce values to prevent replay attacks. Nonce values MUST have appropriate expiration times and MUST be invalidated after successful use. The PID Authorization Server and MRTD PoP Service MUST maintain synchronized nonce validation to ensure session integrity.
+
+Security Controls
+^^^^^^^^^^^^^^^^^
+
+The following security controls MUST be implemented in the protocol:
+
+.. _table_eID_MRTD_Security_Controls:
+.. list-table:: eID Substantial Authentication with MRTD Verification Security Controls
+   :widths: 10 60 30
+   :header-rows: 1
+
+   * - **#**
+     - **Description**
+     - **Phase**
+   * - **SC1**
+     - The network communication channels are protected by using TLS v1.2+ with secure ciphers.
+     - All
+   * - **SC2**
+     - The Wallet Instance ensures the authenticity of the PID Provider and the eID Identity Provider by pinning the leaf certificate of each server.
+     - All
+   * - **SC3**
+     - The PID Authorization Server verifies that the eID Identity Provider used within the *eID LoA3* phase is accredited.
+     - Phase 2
+   * - **SC4**
+     - The PID Authorization Server verifies the authenticity and integrity of the data extracted from the *eID LoA3* assertion, by checking the digital signature.
+     - Phase 2
+   * - **SC5**
+     - The challenge value and all nonce values are generated with safeguards to prevent bruteforce or guessing attacks.
+     - Phase 3
+   * - **SC6**
+     - The PID Provider verifies all the nonce values to detect replay attacks.
+     - Phase 3
+   * - **SC7**
+     - The Wallet Instance verifies that ``challenge_info`` is properly signed by the PID Authorization Server. Moreover, it checks that ``challenge_info`` contains: an ``iss`` value corresponding to the value of the PID Authorization Server; an aud value equal to the ``client_id`` of the Wallet Instance; and a ``state`` value equal to the one in the PAR request, to be sure that the response is bound to the initial request that is made by the Wallet Instance in Step 2. Therefore the information provided as part of ``challenge_info``, in particular the ``htu`` that corresponds to the redirect url to follow for the *MRTD PoP*, is not tampered with.
+     - Phase 3
+   * - **SC8**
+     - The PID Provider checks that the ``auth_session`` is associated with the same Wallet Instance in all the requests within the *MRTD PoP* phase. Therefore the PID Provider can be sure that the Wallet Instance performing the *MRTD PoP* phase: is trusted; is always the same across the protocol; and has previously started the PID issuance (PAR request). This can be implemented by requesting the Wallet Instance to perform a proof of possession of its private key (e.g., within OAuth-Client-Attestation or by signing a nonce value).
```

Comment:
```suggestion
     - The PID Provider checks that the ``auth_session`` is associated with the same Wallet Instance in all the requests within the *MRTD PoP* phase. Therefore the PID Provider checks that the Wallet Instance performing the *MRTD PoP* phase is trusted, it is always the same across the requests, and it has previously started the PID issuance (PAR request). This can be implemented by requesting the Wallet Instance to perform a proof of possession of its private key (e.g., within OAuth-Client-Attestation or by signing a nonce value).
```